stream/address_extractor.js

```diff
@@ -50,42 +50,38 @@ module.exports = function(){
       var type = isNamedPoi ? 'poi-address' : 'address';
       var record;
 
-      try {
-        // copy data to new document
-        record = new Document( 'osmaddress', type + '-' + doc.getType() + '-' + (doc.getId() || ++idOrdinal) )
-          .setName( 'default', doc.address.number + ' ' + doc.address.street )
-          .setCentroid( doc.getCentroid() );
-
-        [ 'name', 'number', 'street', 'zip' ].forEach( function ( prop ){
-          try {
-            record.setAddress( prop, doc.getAddress( prop ) );
-          } catch ( ex ) {}
-        });
+      // accept semi-colon and comma delimited house numbers
+      // ref: https://github.com/pelias/openstreetmap/issues/21
+      var streetnumbers = doc.address.number.split(/[;,]/);
+      for( var i in streetnumbers ){
```

Comment:
Can you use a `.forEach()` here, or at least a conventional for loop (`for(var i = 0; // etc`)?